src/org/omegat/Bundle.properties

```
@@ -2876,6 +2883,13 @@ PREFERENCES_WARNING_NEEDS_RELOAD=Reload the current project to apply these chang
 PREFS_TITLE_PLUGINS=Plugins
 PREFS_PLUGINS_BROWSE_ONLINE=&Browse Online
 PREFS_PLUGINS_AVAILABLE_ONLINE=There are many plugins available for OmegaT online.
+PREFS_PLUGINS_INSTALL_FROM_DISK=&Install plugin from file
+PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION=Install plugin file
+PREFS_PLUGINS_CONFIRM_UPGRADE=Upgrade/override plugin {0}\n\
+   current: {1}\n\
+   new version: {2}
+PREFS_PLUGINS_CONFIRM_INSTALL=Install plugin {0} {1}
```

```suggestion
PREFS_PLUGINS_CONFIRM_INSTALL=Install `{0}` plugin, version `{1}`
```

It is better to insert "version" because when old plugin loaded , version number become "unknown".
When old plugin loaded, it will be like  "Install `example filter` plugin, version `unkown`"